CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[Docusaurus](https://docusaurus.io/) is a static site generator that helps you ship beautiful, accessible docs. For building our documentation engine, we have made certain modifications to the template generated by [Docusaurus](https://docusaurus.io) to be able to properly showcase all our contents with their respective versions.
+
+### Built With
+
+- [Docusaurus](https://docusaurus.io/)
+- [React](https://reactjs.org/)
+- [Tailwind](https://tailwindcss.com/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This section describes how you can get our documentation portal up and running on your machine.
+
+### Prerequisites
+
+- [node](https://nodejs.org/en/)
+- [npm](https://www.npmjs.com/)
+
+### Installation
+
+1. Clone the repo
+
+```shell
+git clone https://github.com/keptn-sandbox/new-keptn-docs-engine
+```
+
+2. Install NPM packages
+
+```shell
+npm install
+```
+
+3. Run the app
+
+```shell
+npm start
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+<!-- In usage, mention how to edit the docs, how to update versions, etc. -->
+
+### Writing Documentation
+
+To just edit older documentation, go to the specified versioned folder for a section, for example, you want to edit documentation for Release 0.16.x, open up `Release 0.16.x` folder and edit the required files there.
```

Comment:
I have not seen the plan for how the source files will be structured.  But our current structure, with some of the docs under 0.16.x directories on the web site is going away, right?

The option of editing files for already-released docs should be a special case.  It will happen but most modifications should be made to the "next" tree